api/python/provisioner/__init__.py

```diff
@@ -78,6 +79,7 @@
     'set_network',
     'set_swupdate_repo',
     'sw_update',
+    'rollback_update',
```

Comment:
Should we change this to
**sw_rollback**

Reply:
addressed